stetho/src/main/java/com/facebook/stetho/inspector/elements/android/AndroidDOMProvider.java

```diff
@@ -129,4 +157,140 @@ public View getHighlightingView(Object element) {
 
     return highlightingView;
   }
+
+  private List<Window> getWindows() {
```

Comment:
nit: because this allocates a new list each time, prefer a non-getter verb like: `collectWindows()`.